desktop/core/src/desktop/conf.py

```diff
@@ -1610,7 +1610,15 @@ def is_gunicorn_report_enabled():
       key="oidc_rp_client_secret",
       help=_("The client secret as relay party set in OpenID provider."),
       type=str,
-      default="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      default=""
+    ),
```

Comment:
@SoniaComp Looks like there are some unit test failures, can you please check? Also is this change required?

Reply:
@SoniaComp I'll try fixing that in sometime.

Reply:
Raised https://github.com/cloudera/hue/pull/4173

Reply:
@Harshg999 I merged the master branch! Thank you for checking my code 🚀